sysbuild/bootloader/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 Nordic Semiconductor
```

Comment:
did you read https://github.com/zephyrproject-rtos/zephyr/issues/40309 ?
> The items below are in the scope of this issue:
>
>    Building once and getting a stitched image that contains both application and bootloader (MCUboot in particular)

so the main use-case for this is actually to be able to build and flash an application together with a bootloader (MCUboot) in one go.

But of course the design of SYSBuild is made in an extendable way so that it can easily be extended to cover more use-case and samples.

Reply:
Yes, I did. And at the beginning: 

>SYSBuild is a higher level CMake and Kconfig based build system which support
configuring and building a Zephyr application together with additional projects,
both Zephyr based but also non-Zephyr based projects.


>But of course the design of SYSBuild is made in an extendable way so that it can easily be extended to cover more use-case and samples.

So can you please make your PR reflect the generic nature?

Reply:
@jfischer-no done.